### ISSUE TITLE
On android device text doesn't print

### ISSUE THREAD

**Issue by GameMasterPDX** [fadeinside/s3air-achievements-plus-tools#1]
It doesn't put text with code on screen

**Comment by Juani12ok** [comment]
1. Download the Windows version of Sonic 3 A.I.R.
2. Extract `config.json`
3. Copy/Move `config.json` onto `/storage/emulated/0/android/data/org.eukaryot.sonic3air/files`
4. Open the file you've just copied/moved
5. Search for `"DevMode": 0,` and change it to `"DevMode": 1,` instead
6. Save the changes
7. Open Sonic 3 A.I.R.

And that's it! You're good to go!

**Comment by GameMasterPDX** [comment]
its been 3 years i dont even play on android anymore

**Comment by Juani12ok** [comment]
Yeah should've expected that, but I'll leave the comment just in case someone else currently has the same issue.

Btw do you remember which version of the game you had back then?

**Comment by GameMasterPDX** [comment]
probably latest android release of the time
